### PR TITLE
Clean up workarounds for compatibility with GHC < 8

### PR DESCRIPTION
**Pull request by arybczak**

### CODE REVIEW

File: src/Database/PostgreSQL/PQTypes.hs [hunk 1]
```diff
@@ -59,6 +59,7 @@ module Database.PostgreSQL.PQTypes (
   ) where
 
 import Data.Functor.Identity
+import Database.PostgreSQL.PQTypes.Interval
```
Comment (23Skidoo) [inline src/Database/PostgreSQL/PQTypes.hs]: This change looks unnecessary. Did the sort imports script do that? Looks like it has a bug.

Comment (arybczak) [reply src/Database/PostgreSQL/PQTypes.hs]: Huh, right. It's only our repo's internal version that has this bug though, I fixed it some time ago: https://github.com/arybczak/haskell-dev-utils/commit/96d0578216a48fe3ba57b07a30bbf44021a6d1af

Comment (arybczak) [reply src/Database/PostgreSQL/PQTypes.hs]: Force pushed version without changes to this file.

Comment (23Skidoo) [reply src/Database/PostgreSQL/PQTypes.hs]: There are similar unnecessary changes to lots of other files, can you just rerun the fixed script on the whole repo perhaps?

**Comment by arybczak** [comment]
Alright, it should be good now.

**Comment by 23Skidoo** [comment]
Looks good now!

**Pull request merged**